user/conditional-builds-stages-jobs.md

```diff
@@ -112,6 +112,29 @@ Known functions are:
 
 The function `env` currently only supports environment variables that are given in your build configuration (e.g. on `env` or `env.global`), not environment variables specified in your repository settings.
 
+#### Right hand side
+
+It is currently not possible to compare function calls. This means that if you try to evaluate something similar to:
+
+`env(PRIOR_VERSION) != env(RELEASE_VERSION)`
+
+where PRIOR_VERSION and RELEASE_VERSION are environment variables defined elsewhere in .travis.yml, the condition will be evaluated as true, even when it is false. This sad state of affairs is being addressed in an upcoming feature release.
+
+A work-around is to create a script which expresses the conditional test in reverse, and applies `travis_terminate` when the condition is false. This script can be called within the Deploy stage.
+For instance, continuing with the above example, the Deploy stage would include:
+
+```
+- stage: deploy
+   if: attribute=value
+   env:
+    - PRIOR_VERSION=$(git describe --abbrev=0 --tags)
+    - RELEASE_VERSION=$(grep SOME MAGIC HERE PLEASE HELP ME TO PUT SOMETHING THAT MAKES SENSE HERE BUT IS ALSO GENERIC)
+   script:
+    - "$PRIOR_VERSION" = "$RELEASE_VERSION" && travis_terminate || echo "Deploying latest version ..."
```

Comment:
I suggest we show an if/then statement instead as it seems more natural to me. e.g.

```
script:
  - if [ $PRIOR_VERSION == $RELEASE_VERSION ]; then travis_terminate 0; fi
  # else
  - echo "Deploying latest version ..."
```

Reply:
Pls ignore this branch. We are working on it elsewhere.

Reply:
Ah! I was just wondering why it was closed.

Let me know if you want to review what your are doing over there. :)